geometric_kernels/kernels/feature_maps.py

```diff
@@ -59,7 +56,7 @@ def _map(X: B.Numeric, params, state, **kwargs) -> B.Numeric:
         )
 
         weights = B.transpose(B.power(spectrum, 0.5))  # [1, M]
-        Phi = state["eigenfunctions"]
+        Phi = kernel.eigenfunctions()
```

Comment:
What about making `eigenfunctions` a property? Calling it as a method seems old-fashioned to me.